proposals/csharp-10.0/parameterless-struct-constructors.md

```diff
@@ -0,0 +1,157 @@
+# Parameterless struct constructors
+
+## Summary
+
+Support parameterless constructors and instance field initializers for struct types.
+
+## Proposal
+
+### Instance field initializers
+Instance field declarations for a struct may include initializers.
+
+As with [class field initializers](https://github.com/dotnet/csharplang/blob/master/spec/classes.md#instance-field-initialization):
+> A variable initializer for an instance field cannot reference the instance being created. 
+
+### Constructors
+A struct may declare a parameterless instance constructor.
+The struct can be declared as `struct`, `readonly struct`, or `ref struct`.
+
+If the struct does not declare a parameterless instance constructor, and the struct has no fields with variable initializers, the struct (see [struct constructors](https://github.com/dotnet/csharplang/blob/master/spec/structs.md#constructors)) ...
+> implicitly has a parameterless instance constructor which always returns the value that results from setting all value type fields to their default value and all reference type fields to null.
+
+If the struct does not declare a parameterless instance constructor, and the struct has field initializers, a `public` parameterless instance constructor is synthesized.
+The parameterless constructor is synthesized even if all initializer values are zeros.
+
+### Modifiers
+A parameterless instance constructor may be less accessible than the struct.
+
+A parameterless instance constructor may be declared `extern`.
+
+### Executing field initializers
+Execution of struct instance field initializers matches execution of [class field initializers](https://github.com/dotnet/csharplang/blob/master/spec/classes.md#instance-variable-initializers):
+> When an instance constructor has no constructor initializer, ... that constructor implicitly performs the initializations specified by the _variable_initializers_ of the instance fields ... . This corresponds to a sequence of assignments that are executed immediately upon entry to the constructor and before the implicit invocation of the direct base class constructor. The variable initializers are executed in the textual order in which they appear in the ... declaration.
+
+### Definite assignment
+Instance fields must be definitely assigned in struct instance constructors that do not have a `this()` initializer (see [struct constructors](https://github.com/dotnet/csharplang/blob/master/spec/structs.md#constructors)).
+
+Definite assignment is required for explicit parameterless constructors as well.
```

Comment:
Think we need to clean up the wording of this section a bit to make it clearer when we're talking about definite assignment within the `struct` constructor and when we're talking about uses of a `struct`